sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -33,6 +33,11 @@
 #include "debug.h"
 
 /**
+ * @brief Helper macro for LL_SEARCH to compare TCBs
+ */
+#define TCB_EQUAL(a,b)      ((a) == (b))
+
+/**
```

Comment:
That is not entirely true, if you look at the actual source [here](https://github.com/troydhanson/uthash/blob/master/src/utlist.h#L441) you'll see that it merely calls whatever you insert as `cmp` like a function with 2 parameters. Hence, the macro should do here, too, which might  be even better as this whole `LL_` stuff is lots of preprocessor magic 😉